apps/avifdec.c

```diff
@@ -231,87 +273,38 @@ int main(int argc, char * argv[])
         return 1;
     }
 
+    if (!inputFilename) {
+        fprintf(stderr, "Missing input filename\n");
+        syntax();
+        return 1;
+    }
+
+    avifAppFileFormat outputFormat = AVIF_APP_FILE_FORMAT_UNKNOWN;
     if (infoOnly) {
-        if (!inputFilename || outputFilename) {
+        if (outputFilename) {
```

Comment:
Could be done for `iccOverrideFilename` but not worth it

Reply:
Ack